### PR TITLE
[stable/prometheus] Update image

### PR DESCRIPTION
**Pull request by naseemkullah**
#### What this PR does / why we need it:

https://www.robustperception.io/new-features-in-prometheus-2-16-0

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by naseemkullah** [comment]
cc @zanhsieh @gianrubio

File: stable/prometheus/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: prometheus
-version: 10.4.0
-appVersion: 2.15.2
+version: 10.4.1
```
Comment (zanhsieh) [inline stable/prometheus/Chart.yaml:3]: I think this should be major version. 10.5.0.

Comment (naseemkullah) [reply stable/prometheus/Chart.yaml:3]: Sounds good, I'll actually open a new PR, something weird about this one and keeps failing the e2e test. 🤷‍♂

**Pull request closed without merging**